types/email-templates/index.d.ts

```diff
@@ -170,7 +177,15 @@ declare class EmailTemplate<T = any> {
      * @param view The Html pug to render
      * @param locals The template Variables
      */
-    render(view: string, locals: T): Promise<string>;
+    render(view: string, locals?: T): Promise<string>;
+    /**
+     * Render all available template files for a given email
+     * template (e.g. `html.pug`, `text.pug`, and `subject.pug`)
+     *
+     * @param view Name of the template
+     * @param locals The template variables
+     */
+    renderAll(view: string, locals?: T): Promise<Partial<EmailMessage>>;
```

Comment:
`renderAll` seems to have 3rd argument of type `Partial<EmailMessage>`. Just as comment, not a review blocker as it is not even documented and looks quite odd anyways.